src/components/soknadsStatus/SoknadsStatus.tsx

```diff
@@ -53,37 +53,41 @@ const SoknadsStatus: React.FC<Props> = ({status, sak, leserData}) => {
+                    const soknadBehandlesIkke = status === SoknadsStatusEnum.BEHANDLES_IKKE;
                     return (
                         <div className="status_detalj_panel" key={index}>
                             <div className={"status_detalj_linje"}>
                                 <div className="status_detalj_panel__tittel">
                                     <Element>{statusdetalj.tittel}</Element>
                                 </div>
-                                {kanVises &&
-                                    saksStatus !== SaksStatusEnum.IKKE_INNSYN &&
-                                    saksStatus !== SaksStatusEnum.BEHANDLES_IKKE &&
-                                    status !== SoknadsStatusEnum.BEHANDLES_IKKE && (
-                                        <div className="status_detalj_panel__status">
-                                            <EtikettLiten>{saksStatus}</EtikettLiten>
-                                        </div>
-                                    )}
+                                {kanVises && !sakIkkeInnsyn && !sakBehandlesIkke && !soknadBehandlesIkke && (
+                                    <div className="status_detalj_panel__status">
+                                        <EtikettLiten>{saksStatus}</EtikettLiten>
+                                    </div>
+                                )}
                             </div>
-
                             {statusdetalj.melding && statusdetalj.melding.length > 0 && (
                                 <div className="panel-glippe-over">
                                     <Normaltekst>{statusdetalj.melding}</Normaltekst>
                                 </div>
                             )}
-                            {(saksStatus === SaksStatusEnum.BEHANDLES_IKKE ||
-                                status === SoknadsStatusEnum.BEHANDLES_IKKE) &&
-                                saksStatus !== SaksStatusEnum.IKKE_INNSYN && (
-                                    <div className="panel-glippe-over">
-                                        <Normaltekst>
-                                            <FormattedMessage id="status.behandles_ikke_ingress" />
-                                        </Normaltekst>
-                                    </div>
-                                )}
-                            {saksStatus === SaksStatusEnum.IKKE_INNSYN && (
+                            {sakIkkeInnsyn && soknadBehandlesIkke && (
+                                <div className="panel-glippe-over">
+                                    <Normaltekst>
+                                        <FormattedMessage id="status.soknad_behandles_ikke_ingress" />
+                                    </Normaltekst>
+                                </div>
+                            )}
+                            {(sakBehandlesIkke || soknadBehandlesIkke) && !sakIkkeInnsyn && (
```

Comment:
Jeg forstår ikke helt. Jeg trodde denne teksten skulle komme dersom: 
`sakBehandlesIkke && !soknadBehandlesIkke` ? 

Altså at: 
```
soknadBehandlesIkke  --> soknad_behandles_ikke_ingress
sakBehandlesIkke && !soknadBehandlesIkke  --> sak_behandles_ikke_ingress
sakIkkeInnsyn && !soknadBehandlesIkke --> ikke_innsyn_ingress
```

En sak skal vel ikke kunne ha både `sakBehandlesIkke` og `sakIkkeInnsyn` samtidig. Så det skal ikke være behov for å ha en sjekk med begge disse?

Reply:
Det er sant, jeg skal ta å endre på det.